main.go

```diff
@@ -110,6 +111,10 @@ func main() {
 
 	ctrl.SetLogger(utils.GetDefaultLogger(logLevel))
 
+	bInfo, _ := debug.ReadBuildInfo()
+	setupLog.Info(bInfo.Main.Path, "version", caphversion.Get().String(),
```

Comment:
where is the advantage of this line compared to the "setupLog.Info("starting manager", "version", caphversion.Get().String())" that we already have?

Reply:
The string of caphversion is empty. You can see that in the testing-cluster:

```
❯ k get -n mgt-system deployments.apps caph-controller-manager -o yaml | grep image:
        image: ghcr.io/syself/caph:v1.0.0
```

```
❯ k logs -n mgt-system deployments/caph-controller-manager   | grep 'starting manager'
Found 4 pods, using pod/caph-controller-manager-5cbd64b89c-4cj8t
{"level":"INFO","time":"2025-03-16T03:56:44.057Z","logger":"setup","file":"./main.go:237","message":"starting manager","version":""}

```

With Go1.24 the string will be very useful, because it contains the git-hash if the build is not clean.

Reply:
I don't get why you added a second version of the same code @guettli .. Sorry. Can you explain it for dumb people? 
I would in general expect that if you write code that replaces old code, that you would also remove this old code and don't keep it...

Reply:
I removed the part you mentioned. We will do that via #1553